lib/klaviyo/apis/track.rb

```diff
@@ -0,0 +1,32 @@
+module Klaviyo
+  class Track < Client
+    def self.track(event, kwargs = {})
+      defaults = {:id => nil, :email => nil, :properties => {}, :customer_properties => {}, :time => nil}
+      kwargs = defaults.merge(kwargs)
+
+      if kwargs[:email].to_s.empty? and kwargs[:id].to_s.empty?
+        raise KlaviyoError.new('You must identify a user by email or ID')
+      end
+
+      customer_properties = kwargs[:customer_properties]
```

Comment:
Would it make sense to move this into a method and combine the track/identify files given both have the same flow? Probably have different methods to make the identify and track calls?